styles.css

```diff
@@ -0,0 +1,776 @@
+    font-size: 1.3rem;
+    font-weight: 300;
+    margin-bottom: 0.5rem;
+  }
+
+  .description-main-container {
+    color: #fff;
+    height: 6rem;
+    padding-left: 0.5rem;
+    padding-right: 0.5rem;
+    font-size: 1.1rem;
+    font-weight: 200;
+  }
+
+  .speakers {
+    font-size: 2rem;
+  }
+
+  .partners {
+    width: 100%;
+    border: 2px solid red;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #414246;
+    margin-bottom: 5rem;
+  }
+
+  .partner-title {
+    text-align: center;
+    padding-top: 1rem;
+    color: #7d7d7d;
+    font-size: 3rem;
+  }
+
+  .partner-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    justify-items: center;
+    align-items: center;
+    margin-bottom: 5rem;
+  }
+
+  #airbnb {
+    padding-top: 1rem;
+  }
+
+  .partner-image {
+    width: 18%;
+    gap: 1rem;
+  }
+
+  .footer-project {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 3rem;
+    width: 100%;
+    margin-bottom: 3rem;
+  }
+
+  .footer-container-desktop {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 80%;
+  }
+
+  .footer-logo {
+    width: 10rem;
+    height: 8rem;
+  }
+
+  .license {
+    height: 3rem;
+    font-size: 1.3rem;
+    width: 32rem;
+    padding-top: 1.2rem;
+  }
+
+  .main-content-about {
+    margin-top: 7rem;
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    align-items: center;
+  }
+
+  .main-section-about-title {
+    color: #3e3e3e;
+  }
+
+  .breaklineabout {
+    border: 2px solid #f65d14;
+    width: 3.5%;
+  }
+
+  .concert-paragraph-about {
+    font-size: 1.3rem;
+    text-justify: auto;
+  }
+
+  .email-about {
+    margin-bottom: 6rem;
+  }
+
+  .about-logo {
+    width: 60%;
+    height: 30rem;
+  }
+
+  .past-global-title {
+    color: #3e3e3e;
+  }
+
+  .edition-cards {
+    display: flex;
+    flex-direction: row;
+    gap: 2rem;
+  }
+
+  .partners-about {
+    display: none;
+  }
+
+  .footer-about {
+    display: flex;
+    background-color: #2e2e2e;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 3rem;
+    height: 20rem;
+    width: 100%;
+    margin-bottom: 3rem;
+  }
+
+  .footer-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    width: 80%;
+  }
+
+  .footer-logo-about {
+    width: 10rem;
+    height: 8rem;
+  }
+
+  .license-about {
+    height: 5rem;
+    font-size: 1.5rem;
+    width: 32rem;
+    color: #fff;
+    padding-top: 1.2rem;
+  }
+
+  .speakers-container {
+    display: grid;
+    grid-template-columns: repeat(2, 50%);
+    justify-content: center;
+    gap: 2rem;
+    width: 90%;
+    margin-bottom: 7rem;
+  }
+}
```

Comment:
- Your is generally clean, and I love this! :love_you_gesture: but I'd like to suggest that you create a class for each `css property` that is used more than ones so you only declare it once and then allow your class to act as a reference. I see different `flex` properties and `font` properties being repeated more than twice. This will make your code DRY and cleaner with shorter code.

Reply:
will solve in the future